### ISSUE TITLE
Feature Request: More actions on windows

### ISSUE THREAD

**Issue by ttencate** [ttencate/taekwindow#98]
Converted from [SourceForge issue 2969929](http://sourceforge.net/support/tracker.php?aid=2969929), submitted by nobody

'would be great if taekwindow allowed window closing, minimizing and maybe also shading (rollup in titlebar). I have too many buttons on my mouse to know what to do do with them and I hate having to go all the way up to the corner of the windows to close them (and let's face it, alt+f4 isn't that comfortable)

**Comment by ttencate** [comment]
Submitted by None

And also temporary transparency. For example,  alt+middlemouse = set transparency to 20% and back to 100% when released. This would allow you to see through windows very quickly. Quite ultimate imo.

**Comment by ttencate** [comment]
Submitted by wanderer51

Add to those requests the ability to make a window to always stay on top and TW will be great.

**Comment by Doryforos** [comment]
Ahem...
1. Customizable hotkeys to toggle horizontal & vertical maximization, maximization & restoration, and to minimize the application; also provide said functions in the application windows control menu, be it by right-clicking the titlebar, or by clicking the window icon, the taskbar button of the application, or its tray icon (if sent to tray).
2. "Taskbar Shuffle" for XP & Vista.

Thanks for your patience! :-)

**Comment by Doryforos** [comment]
Oh!

And the ability to snap application windows to screen edges... bitte!

**Comment by ttencate** [comment]
Yes, when those features are provided by hotkeys, they should also be shown in a more discoverable way. Optional, of course, because not everyone likes to have their title bars and menus cluttered with items they never use.

It is unlikely that I'll spend time on taskbar shuffling. This is now provided by Windows 7, and for those outdated Windows versions there's already a program that provides this functionality just fine, right?

**Comment by Doryforos** [comment]
Hello

Thanks for considering my suggestions!

<rant>Yes, there are a few MS-Win shell enhancements that provide for re-arranging the taskbar buttons, but, it would be nice to (eventually) make Taekwindow an all-encompassing window manager for the MS platform; a little less hand-holding for long-time power users please!</rant> [I guess I got a bit carried away by the name you have given to your application; it's not a bad pun at all, BTW -- I absolutely _love_ it, combined with the link to "WTF" in your FAQ as well, as much as unintented this might be!]

Kindly consider -- it would save valuable system tray space, after all!

Cheers, thanks again!
